draft-ietf-quic-manageability.md

```diff
@@ -1023,6 +1025,53 @@ not arrive at their destination. Admitting a few packets allows the QUIC
 endpoint to determine that the path accepts QUIC. Sudden drops afterwards will
 result in slow and costly timeouts before abandoning the connection.
 
+## Guiding Path MTU
+
+Some networks support 1500-byte packets, but can only do so by fragmenting at a
+lower layer before traversing a smaller MTU segment, and then reassembling.
+This is permissible even when the IP layer is IPv6 or IPv4 with the DF bit set,
+because it occurs below the IP layer. However, this process can add to compute
+and memory costs, leading to a bottleneck that limits network capacity. In such
+networks this generates a desire to influence a majority of senders to use
+smaller packets, so that the limited reassembly capacity is not exceeded.
+
+For TCP, MSS clamping ({{Section 3.2 of RFC4459}}) is often used to change
+the sender's maximum TCP segment size, but QUIC requires a different approach.
+{{Section 14 of QUIC-TRANSPORT}} advises senders to probe larger sizes using
+Datagram Packetization Layer PMTU Discovery ({{?DPLPMTUD=RFC8899}}) or Path
+Maximum Transmission Unit Discovery (PMTUD: {{?RFC1191}} and {{?RFC8201}}).
+This mechanism will encourage senders to approach the maximum size, which
+could cause fragmentation with a network segment that they may not be aware of.
+
+If path performance is limited when sending larger packets, an on-path
+device should support a maximum packet size for a specific transport flow
+and then consistently drop all packets that exceed the configured size
+when the inner IPv4 packet has DF set, or IPv6 is used. Endpoints can cache
+PMTU information between IP flows, in the IP-layer cache, so short-term
+consistency between the PMTU for flows can help avoid an endpoint using a
+PMTU that is inefficient.
+
+If network operators wish to start to drop large packets to avoid
+fragmentation, they may wish to focus on QUIC. QUIC packets cannot always
+be identified ({{sec-identifying}}), but we assume that a set of candidate
+QUIC flows have been chosen. For QUIC endpoints using DPLPMTUD it is
+minimally sufficient for the path to silently drop a packet larger than the
+supported size. (A QUIC probe packet is used to discover the PMTU. If lost,
+this does not impact the flow of QUIC data.)
```

Comment:
```suggestion
If network operators wish to drop large packets to avoid
fragmentation, they may wish to focus on QUIC. QUIC packets cannot always
be identified ({{sec-identifying}}), but we assume that a set of candidate
QUIC flows have been chosen. For QUIC endpoints using DPLPMTUD it is
minimally sufficient for the path to silently drop a packet larger than the
supported size. (A QUIC probe packet is used to discover the PMTU. If lost,
this does not impact the flow of QUIC data.)
```